### PR TITLE
chore(website): fix parsing in function list

### LINKED ISSUE #2486: Documentation display errors
screen shot 2015-09-10 at 8 40 41 pm

For some reason there seem to be a lot of dead tags floating around in the docs. Almost looks like it's not rendering fully.

Google Chrome Version 45.0.2454.85 (64-bit) on Mac OSX

### PR DESCRIPTION
**Pull request by sjelin**
fixes #2486 and #2672

### CODE REVIEW

**Comment by juliemr** [comment]
Don't we have to include `ngSanitize` in the app to use `ng-bind-html`?

**Comment by sjelin** [comment]
Oh, probably.  I was coping code from another part of the website but I guess that was foolish

**Comment by sjelin** [comment]
Fixed

**Comment by sjelin** [comment]
Now also fixes #2672!  Also, we should talk about XSS

File: website/js/shared.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/*
+ * Use the $sce service to trust the html rendered in the view.
+ * Also parse links
+ *
+ * @param $sce
+ * @param html
```
Comment (juliemr) [inline website/js/shared.js]: add `@return` doc

**Comment by juliemr** [comment]
Can you add a test for `trust` to https://github.com/angular/protractor/blob/master/website/test/unit/api-controller-spec.js?

**Comment by sjelin** [comment]
All comments addressed

**Pull request merged**